src/css/imports/transitions.less

```diff
@@ -0,0 +1,55 @@
+.jwplayer.jw-skin-seven {
```

Comment:
Start this file by defining which classes will make element transition which properties:

```css
.jw-controlbar,
.jw-controls-right,
.jw-display,
.jw-overlays {
    transition: opacity visibility 1s;
}
````

Then you don't have to add `transition` anywhere else, just change the property that will be transitioned. There doesn't need to be any specificity here because `transition` isn't likely to be overridden.